test/e2e/e2e.go

```diff
@@ -127,6 +134,28 @@ var _ = ginkgo.Describe("[efs-csi] EFS CSI", func() {
 		fileSystemId = id
 	})
 
+	var once sync.Once
```

Comment:
This synchronization turned out to be pointless because the other BeforeEach/AfterSuite blocks are not yet synchronized, so running tests in parallel usually fails with "AlreadyExists" type errors anyway. But I'm going to leave it here, I'm working on another PR to fix it. The CI runs tests serially so it makes no difference at the moment.